bundlebee-documentation/src/main/minisite/content/getting-started.adoc

```diff
@@ -118,38 +119,50 @@ spec:
         - name: hello
           image: hello-world:latest
           envFrom:apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: hello-kubernetes
-spec:
-  replicas: 3
-  selector:
-    matchLabels:
-      app: hello-kubernetes
-  template:
-    metadata:
-      labels:
-        app: hello-kubernetes
-    spec:
-      containers:
-      - name: hello-kubernetes
-        image: paulbouwer/hello-kubernetes:1.8
-        ports:
-        - containerPort: 8080
 ----
 
 Now you can install your alveolus: `mvn install`.
 
 This will create a `target/my-app-alveolus-1.0-SNAPSHOT.jar` file which is the entry point to deploy your application.
 
-Now you can deploy this executing `bundlebee apply --from com.company:my-app-alveolus:1.0-SNAPSHOT` --descriptor hello.
+Now you can deploy this executing `bundlebee apply --from com.company:my-app-alveolus:1.0-SNAPSHOT --alveolus hello`.
 
 And that's it, now hello world application should be up and running in your Kubernetes cluster.
 
 TIP: it is more than encouraged to name the kubernetes descriptor based on the project unique identifier.
 For example `<groupId>.<artifactId>.<alveolus_name>.yaml`.
 It will avoid conflicts when multiple alveolus are in the same classpath using native java mode and not file driven mode.
 
+You could also have used the `bundlebee-maven-plugin`, simply add it in the plugins section of the pom.xml :
+
+[source,`xml`]
+.pom.xml
+----
+<plugins>
+    <plugin>
+      <groupId>io.yupiik</groupId>
+      <artifactId>bundlebee-maven-plugin</artifactId>
+      <version>1.0.11</version>
+      <executions>
+        <execution>
+          <id>hello-world</id>
+          <phase>none</phase>
+          <goals>
+            <goal>apply</goal>
+          </goals>
+          <configuration>
+            <verbose>true</verbose>
+            <manifest>${project.basedir}/src/main/resources/bundlebee/manifest.json</manifest>
```

Comment:
do we want to add a comment (admonition ;)) to say it is because it is an exploded folder and that for packaged (jar) alveolus it is replaced by the location or is it complex enough for the sample?